source/code/plugin/in_kube_podinventory.rb

```diff
@@ -171,6 +171,7 @@ def parse_and_emit_records(podInventory, serviceList)
               containerRestartCount = container['restartCount']		
               record['ContainerRestartCount'] = containerRestartCount
               containerStatus = container['state']
+              record['ContainerNotRunningReason'] = ''
```

Comment:
Rename something like 'ContainerStatusReason' so that in the future if we want to do reason for non 'not-running' status, we can use the same field?

Reply:
updated to 'ContainerStatusReason'